app/controllers/claims/support/schools/grant_conditions_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Claims::Support::Schools::GrantConditionsController < Claims::Support::ApplicationController
+  include Claims::BelongsToSchool
+
+  before_action :authorize_support_user
+
+  def remove; end
+
+  def destroy
+    @school.remove_grant_conditions_agreement
+    redirect_to claims_support_school_path(@school), flash: { success: t(".success") }
+  end
+
+  private
+
+  def authorize_support_user
+    authorize @support_user || Claims::SupportUser
+  end
+end
```

Comment:
This is a good way to structure the "grant conditions acceptance" as a resource, but in this case, especially as we're only mutating a School record, we can do:

```ruby
class Claims::Support::SchoolsController
  # PUT /support/schools/:school_id/remove_grant_conditions_acceptance
  def remove_grant_conditions_acceptance
    # ...
  end
end
```

`PUT` instead of `DELETE` as we're mutating the School row, not dropping any rows.